ccm_web/client/src/utils/canvasSectionNameValidator.ts

```diff
@@ -19,10 +19,19 @@ class DuplicateSectionNameValidator implements IValidator {
   }
 }
 
+class SectionNameTooLongValidator implements IValidator {
+  validate = (sections: CanvasCourseSection[], sectionName: string): ICanvasSectionNameInvalidError | undefined => {
+    if (sectionName.length > 255) {
+      return { reason: 'Section name too long.' }
```

Comment:
I feel somebody putting a section name greater that 255 will never happen!  My simple suggestion here would be can we word this as  `Section name should be 255 characters or less`. Same text change goes above

Reply:
Good call.